frontend/.env

```
@@ -0,0 +1,2 @@
+VITE_API_URL=http://localhost:8888
+VITE_APP_VERSION="local-v1.0.0"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)